test/integration/h1_capture_fuzz_test.cc

```diff
@@ -0,0 +1,79 @@
+#include <functional>
+
+#include "common/common/assert.h"
+#include "common/common/logger.h"
+
+#include "test/fuzz/fuzz_runner.h"
+#include "test/integration/capture_fuzz.pb.h"
+#include "test/integration/http_integration.h"
+#include "test/test_common/environment.h"
+
+namespace Envoy {
+
+class H1FuzzIntegrationTest : public HttpIntegrationTest {
```

Comment:
This is really cool, I wonder if we would end up getting more frontend coverage if we also had a test that didn't include upstream at all. Basically, just setup a config with router with a catch-all route that returns a 200 direct response. Then basically just assert that we always get some type of HTTP response code? I feel like that would have less timing stuff to deal with.